Test/Applications/test_xia2setup.py

```diff
@@ -0,0 +1,39 @@
+import mock
+import os
+import sys
+from importlib import reload
+
+from xia2.Handlers.XInfo import XInfo
+
+
+def test_write_xinfo_split_sweep(dials_data, tmp_path):
```

Comment:
Could consider writing this test to execute `xia2setup` via procrunner, except for the fact that that is now gone - yes this is probably a reasonable approach to executing the tests. Can extend this as well to cover other use cases described in the issue this fixes.